lib/DxilPIXPasses/DxilDbgValueToDbgDeclare.cpp

```diff
@@ -399,12 +419,217 @@ static OffsetInBits GetAlignedOffsetFromDIExpression(
   return Exp->getBitPieceOffset();
 }
 
+llvm::DISubprogram* GetFunctionDebugInfo(llvm::Module& M, llvm::Function* fn) {
+  auto FnMap = makeSubprogramMap(M);
+  return FnMap[fn];
+}
+
+GlobalVariableToLocalMirrorMap
+GenerateGlobalToLocalMirrorMap(llvm::Module &M,
+                                    llvm::DIGlobalVariable *DIGV) {
+  hlsl::DxilModule &DM = M.GetOrCreateDxilModule();
+  auto entryPoints = DM.GetExportedFunctions();
```

Comment:
So internal functions don't need to add the global mirror?
We might need it if noinline is marked.

Reply:
Good point. I made changes to operate on all functions.